### ISSUE TITLE
is this project still going?

### ISSUE THREAD

**Issue by drcmda** [notrabs/use-ammojs#1]
looks really really interesting! would you be interested to make this pmndrs official with more help and resources?

**Comment by notrabs** [comment]
yes, I am still looking to solve the problem of more capable physics!

I also wouldn't mind this project getting more love at all, or making the (npm) name available for a better implemetation of the idea.

Out of curiosity what is your motivation for considering ammo? Mainly asking because there are also Javascript PhysX bindings available, which I probably would have considered more closely if I knew about them earlier.

**Comment by Glavin001** [comment]
@drcmda @notrabs : What are next steps to make this "pmndrs official" with more support 😃 

I'm loving [use-cannon](https://github.com/pmndrs/use-cannon) so far and would appreciate the support for soft bodies and other features provided by Ammo (Bullet) over Cannon-ES.

I'd also be interested in contributing, with my experience in JavaScript, TypeScript, React, and recently Three.js. Thank you both for building such awesome projects! 🎉 

> there are also Javascript PhysX bindings available, which I probably would have considered more closely if I knew about them earlier.

@notrabs : Are you referring to https://www.npmjs.com/package/physx-js ?

**Comment by drcmda** [comment]
@notrabs sorry i must have missed the notification, too much going on. the main problem with cannon is performance, it just doesn't keep up as the amount of items increases and that seems to be less of an issue with ammo. if these libraries are being developed on poimandres it might help to attract more devs, give it more attention, you can add your donation channels which might see more people chipping in, etc.

**Comment by Glavin001** [comment]
I’d love to contribute to this project, potentially this weekend. Would you be open to reviewing PRs or is this project no longer being maintained and should be forked? Regardless, thanks for getting this started!

**Comment by Glavin001** [comment]
**Update**: I've made progress updating `use-ammojs` to be a more feature complete alternative to `use-cannon`, such as adding [constraints API](https://use-ammojs-git-feat-examples-glavin001.vercel.app/docs/modules.html#useConstraint) and much, much more.

Check out:
- Examples/Demos: https://use-ammojs-git-feat-examples-glavin001.vercel.app/
- API Docs: https://use-ammojs-git-feat-examples-glavin001.vercel.app/docs/index.html
- _Super rough_ work-in-progress Pull Request: https://github.com/Glavin001/use-ammojs/pull/1

I need this for my own project switching from Cannon-ES to Ammo.js so I've temporarily published to [`@glavin001/use-ammojs`](https://yarnpkg.com/package/@glavin001/use-ammojs)

After using it for the past few weeks, I do think `use-ammojs` would be an awesome alternative to `use-cannon` for 3D physics!
@drcmda @notrabs : Have you given any more thought to migrating the project to be [pmndrs](https://github.com/pmndrs) officially supported? 🎉  I'd love to continue contributing with the support of others.

**Comment by Glavin001** [comment]
**Progress Update:** I'm still working on improvements/adding features to `use-ammojs` such that I can replace `use-cannon` in my projects. Once the essential features & performance are good, I'll polish [the Pull Request](https://github.com/Glavin001/use-ammojs/pull/1/commits) and let you know it is ready for review.

The latest, I've added `InstancedMesh` support:
- Instanced: https://use-ammojs-git-feat-examples-glavin001.vercel.app/stress?size=10&height=10&instanced=true
- Not Instanced: https://use-ammojs-git-feat-examples-glavin001.vercel.app/stress?size=10&height=10&instanced=false

You can edit the query parameters to push Ammo.js & Three.js to the limits. On my machine 1000 boxes ☝️ was sufficient to show how `InstancedMesh` is superior. 

It is impressive how powerful Three.js & Ammo.js are together!
A couple more examples:
- https://use-ammojs-git-feat-examples-glavin001.vercel.app/stress?size=5&height=100&instanced=true
- https://use-ammojs-git-feat-examples-glavin001.vercel.app/stress?size=20&height=10&instanced=true

**Comment by fgarrec0397** [comment]
@Glavin001 @drcmda @notrabs do we have any update on the migration status regarding this project becoming pmndrs official?

**Comment by Glavin001** [comment]
I recommend https://github.com/pmndrs/react-three-rapier now. I hear Rapier is faster than Bullet/Ammojs and Rapier has official support for WASM.

**Comment by DeveeiveD** [comment]
tbf rapier doesn't feel as snappy as ammojs
